src/App.js

```diff
@@ -13,6 +14,7 @@ class App extends Component {
         movies: []
       },
       showDetails: false,
+      error: '',
       movieDetails: {
```

Comment:
Feel free to remove this mock data since you've got the fetch for a single movie now.